source/search_page/assets/searchRecipe.js

```diff
@@ -0,0 +1,114 @@
+class SearchRecipe extends HTMLElement {
+    constructor() {
+
+      super();
+      let shadow = this.attachShadow({mode: 'open'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8414003)
```suggestion
      let shadow = this.attachShadow({mode: "open"});
```